### PR TITLE
Fixes#760Update index.html

### LINKED ISSUE #769: Fix color of buttons
### Expected results
The right button should be red like all buttons
### Actual results
screen shot 2018-01-03 at 12 04 47 pm
also increase some space here
screen shot 2018-01-03 at 12 09 03 pm

### Steps to reproduce the issue
Add necessary classes
## Would you like to work on this
Yes I am

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes #769
Changed the color of the button to match the rest 
Before:
screen shot 2018-01-03 at 12 04 47 pm

After:
screen shot 2018-01-03 at 12 13 18 pm

Also added some spacing 
Before: 
screen shot 2018-01-03 at 12 09 03 pm

After:
screen shot 2018-01-03 at 12 13 28 pm
Deploy Link:

https://arjunpandey2003.github.io/gci17.fossasiapcd.org/

### CODE REVIEW

**Comment by saarthakchats** [comment]
I think the old color was looking nice.
The spacing change is good

**Comment by arjunpandey2003** [comment]
@saarthakchats actually the buttons in the other slider are also red so it looks a bit weird being the odd one out

**Comment by arjunpandey2003** [comment]
If you want I could change the color of the second buttton in the second slide two match the earlier color combination.

**Review by nzec — CHANGES_REQUESTED** [review]
The previous buttons were looking better.

And don't use the `<br>` tag for spacing as it is deprecated. Use CSS margin.

**Review by saarthakchats — APPROVED** [review]
That was my only concern. If the other reviewers are okay with it, then it is fine. Rest looks good

**Comment by nzec** [comment]
@arjunpandey2003 I still don't approve the button change, let's wait for the reviewers.

**Comment by arjunpandey2003** [comment]
@duttaditya18 sure my only concern is that in one slide both the buttons are red and in the other one is red and one is transparent

**Comment by arjunpandey2003** [comment]
If you want I could also make the one button in the slide transparent to match the color combination.

**Comment by nzec** [comment]
@arjunpandey2003 Oh! I see. I would actually recommend you to change the one which has both red buttons to one red and one white.

However, it'd be better if you wait for reviewers.

**Comment by arjunpandey2003** [comment]
@duttaditya18 ok sure. Thanks!

**Review by nzec — APPROVED** [review]
Awesome Work! 👍

File: css/custom.css [hunk 1]
```diff
@@ -250,3 +250,6 @@ span.student-project-name {
 #student-blog .col-md-3 a {
 	width: 240px;
 }
+.need{
```
Comment (vaibhavsingh97) [inline css/custom.css]: PLease add space between two classes

File: index.html [hunk 2]
```diff
@@ -228,7 +228,7 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 		</div>
 		<div class="row">
 			<div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 text-center">
-				<a class="mb0 btn btn-lg btn-filled" href="https://gci16.fossasia.org/#stprojects">Student Projects'16</a>
+				<a class="mb0 btn btn-lg btn-filled need" href="https://gci16.fossasia.org/#stprojects">Student Projects'16</a>
```
Comment (vaibhavsingh97) [inline index.html]: please use something relevant class name instead of `need`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (nzec) [reply index.html]: 😄

**Comment by arjunpandey2003** [comment]
Squashing

**Review by codedsun — APPROVED** [review]

**Comment by arjunpandey2003** [comment]
@codedsun can you merge?

**Comment by codedsun** [comment]
@arjunpandey2003 let others mentors also review 👍

**Comment by arjunpandey2003** [comment]
sure

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**